tests/frontend/onnx/test_build_ir.py

```diff
@@ -187,6 +187,28 @@ def test_visit_graph_sub():
     )
 
 
+def test_visit_graph_matmul():
+    # initialize context
+    ctx = OnnxXdslMapping()
+
+    # create graph composed only of one MatMul operation
+    graph, _ = _create_graph_binary_op("MatMul", "matmul_graph")
+
+    # run visit graph
+    visit_graph(graph, ctx)
```

Comment:
can you please run visit_node instead, like in the other tests? Can you please also add a `.verify()` to make sure that the result is correct?

Reply:
Done

Reply:
Thank you! I'm not sure we need this second test, the one above seems to be sufficient to me, am I missing something?